src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -355,10 +355,9 @@ public function deleteBasketBasketelementsAction($basketId, $elementId)
         }
 
         $view = \FOS\RestBundle\View\View::create($basket);
-        $serializationContext = SerializationContext::create();
-        $serializationContext->setGroups(array('sonata_api_read'));
-        $serializationContext->enableMaxDepthChecks();
-        $view->setSerializationContext($serializationContext);
+        $context = new Context();
```

Comment:
Take a look at this: https://github.com/sonata-project/SonataMediaBundle/blob/3.x/Controller/Api/MediaController.php#L410